webpack/assets/javascripts/react_app/ReactApp/ReactApp.js

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { ConnectedRouter } from 'connected-react-router'
 import history from '../history';
-
+import store from '../redux';
 import Layout from '../components/Layout';
 import AppSwitcher from '../routes';
 
 const ReactApp = ({ data: { layout } }) => (
-  <Router history={history}>
-    <Layout data={layout}>
-      <AppSwitcher />
-    </Layout>
-  </Router>
+  <Provider store={store}>
```

Comment:
Are you sure we need it? Don't we wrap everything with the `Provider` when registering them? See: https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/componentRegistry.js#L80-L82

Reply:
the mounter uses this registry, no? 
our react root component doesn't use it